Enemies/Enemy.gd

```diff
@@ -82,15 +84,19 @@ func move_along_path(delta: float, lookat: bool = false) -> void:
 	move_and_slide(velocity, Vector3.UP)
 
 # Find the nearest node to take cover at
+# Will try to get nearest node that is at a distance to player > MINIMUM_RANGE
+# Return absolute nearest node if none
 func get_shortest_node():
 	var coverNodes = get_tree().get_nodes_in_group(
 			"navnodes_not_seen_by_player"
 		)
 	var shortestNodePathDistance = INF
 	var shortestNodePathIndex = null
 	var currentNodePathIndex = 0
+	var minimumRangeNodePathIndex = null
+	var minimumRangeNodePathDistance = INF
 	for n in coverNodes:
-		if n.occupied == true:
+		if n.occupied == true || n.name == "NavNodeSniper":
```

Comment:
This won't work if the sniper node has a name like "NavNodeSniper2" due to there being multiple sniper nodes. This can be removed if the sniper nodes are entirely removed from the normal navnodes group.